cmd/commands/root.go

```diff
@@ -0,0 +1,11 @@
+package commands
+
+import (
+	"os"
+
+	"github.com/tendermint/tmlibs/log"
+)
+
+var (
+	logger = log.NewTMLogger(log.NewSyncWriter(os.Stdout)).With("module", "main")
```

Comment:
This is not duplicated.  Just in the commands dir.  There is no reference to logger in main.go at all.  At least not anymore...